client/src/components/PropertiesList.tsx

```diff
@@ -30,76 +67,174 @@ const PropertiesListWithoutI18n: React.FC<
   }
 > = (props) => {
   const { i18n } = props;
+  const { width: windowWidth, height: windowHeight } = Helpers.useWindowSize();
   const locale = (i18n.language as SupportedLocale) || "en";
 
   const addrs = props.state.context.portfolioData.assocAddrs;
   const rsunitslatestyear = props.state.context.portfolioData.searchAddr.rsunitslatestyear;
+
+  const lastColumnRef = createRef<any>();
+  const isLastColumnVisible = Helpers.useOnScreen(lastColumnRef);
+  /**
+   * For older browsers that do not support the `useOnScreen` hook,
+   * let's hide the dynamic scroll fade by default.
+   */
+  const isOlderBrowser = typeof IntersectionObserver === "undefined";
+  /**
+   * Let's hide the fade out on the right edge of the table if:
+   * - We've scrolled to the last column OR
+   * - We're using an older browser that cannot detect where we've scrolled
+   */
+  const hideScrollFade = isOlderBrowser || isLastColumnVisible;
+
+  const tableRef = useRef<HTMLDivElement>(null);
+  const isTableVisible = Helpers.useOnScreen(tableRef);
+
+  const [headerTopSpacing, setHeaderTopSpacing] = useState<number | undefined>();
```

Comment:
Maybe add a comment here explaining why we're tracking this, and why it's important, from an end-user point of view?